### DISCUSSION TITLE
-path M A

### DISCUSSION THREAD

**Discussion by summer430** [ImageMagick/ImageMagick#2922] [Help]
Hi can someone explain how to use -path M A what does M 0,0 means and for the  A 1,1 0  10,0 10,200 pls explain each number thanks a lot guys

**Comment by snibgo** [comment]
See https://www.w3.org/TR/SVG/paths.html

"M" is "moveto", parameters (x,y), move to X,Y coordinates.

"A" is ellitical arc. The parameters are: (rx ry x-axis-rotation large-arc-flag sweep-flag x y).

**Reply by summer430** [reply]
Thank you for this Im going to study about this 👍

**Comment by fmw42** [comment]
See https://imagemagick.org/Usage/draw/#paths
